http/src/main/resources/init-resources/cluster-site-gce.conf

```diff
@@ -0,0 +1,77 @@
+<VirtualHost _default_:${SSL_HTTPD_PORT}>
```

Comment:
I guess we need a separate version of this file because of the docker networking change, right?

Reply:
yep..the hostnames are different...